example/boing.cpp

```diff
@@ -595,7 +596,7 @@ int main(void) {
     window->callbacks()->on_cursor_move = &cursor_position_callback;
 
     window->makeContextCurrent();
-    gladLoadGLLoader((GLADloadproc) &vkfw::getProcAddress);
+    gladLoadGL((GLADloadfunc) &vkfw::getProcAddress);
```

Comment:
These are my bad 😅.

That being said, since they are not related to CMake, please move them into a separate commit.